app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/websocket/RxWebSocketClient.kt

```diff
@@ -24,18 +24,20 @@ class RxWebSocketClient {
                 }
 
     fun events(serverUri: URI): Observable<Event> =
-        requireActiveClient(serverUri = serverUri).events().flatMap { event ->
-            when (event) {
-                is Event.Close -> {
-                    Timber.i("Received close event, expect restart.")
-                    Observable.timer(3, TimeUnit.SECONDS)
-                        .flatMap { events(serverUri = serverUri) }
-                        .startWith(event)
+        requireActiveClient(serverUri = serverUri)
+            .events()
+            .flatMap { event ->
+                when (event) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/feb0c51d1649c186c776faa50e562320ab9dc395/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/websocket/RxWebSocketClient.kt#L30"></span>Prefer using ‘if’ for binary conditions instead of ‘when’.</td>
    </tr>
  </tbody>
</table>